app/views/documents/archive.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if current_user.assessor? %>
+<% if @planning_application.can_validate? %>
```

Comment:
We can remove this `if` altogether, seeing as we forbid seeing this page if this is false.